### DISCUSSION TITLE
How do we use the files in the Releases section?

### DISCUSSION THREAD

**Discussion by Somu76** [Yvand/EntraCP#259] [General]
I am seeing a number of zip files in addition to the Assembly-Bindings-config and the WSP file. Do we need to extract the contents of the dependencies zip file anywhere on the server? and what about the nuget zip?

The install steps do not appear to reference these, so looking for guidance.

Using this in SP2019 on a Windows 2016 server, for reference. (Framework - 4.8)

**Comment by Yvand** [comment]
@Somu76 you can ignore the other zip files, which are mostly for debugging

**Reply by Somu76** [reply]
Thanks much for the quick response. 
I was trying to find the assemblies referenced in the assemblybindings config runtime section on one of the SharePoint Servers and am not finding any of them in there. In looking at these on my personal machine, I am seeing these as possibly coming from .NET Core, or .NET SDK, neither of which are present on my SharePoint servers. Given this scenario, is the runtime even needed to be added to the machine.config here? and does their absence indicate that we are missing pre-requisites further?

**Reply by Yvand** [reply]
The only prerequisites on the SharePoint servers is to have the .NET Framework 4.8 (or newer) installed.
All the external assemblies required by EntraCP, including the ones referenced in the machine.config, will be added to the GAC during the deployment of the wsp solution (cmdlet `Install-SPSolution`)
